### ISSUE TITLE
Executing commands freezes game

### ISSUE THREAD

**Issue by while-loop** [MitchBarnett/wiki-bank-tag-integration#4]
The wiki api call is done on the client thread so the game freezes until the request is finished.

I think you could move the request to a new thread or call 
```java
RuneLiteAPI.CLIENT.newCall(request).enqueue(new Callback.....);
```
instead of
```java
RuneLiteAPI.CLIENT.newCall(request).execute();
```

https://www.baeldung.com/guide-to-okhttp#asynchronous-get-with-okhttp

**Comment by MitchBarnett** [comment]
Yes I am aware of this and did plan to make it async I just never got around to it. 

It is only a minor inconvenience as the queries should be quick and I don't think anyone will be running the commands during a time critical activity however, it will be fixed if I push another update.

**Comment by while-loop** [comment]
Right on! It's not a major issue for me either. 

Thanks for the cool plugin 😁
